src/components/titlepane/TitlePane.ts

```diff
@@ -0,0 +1,81 @@
+import { v } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+
+import * as css from './styles/titlePane.css';
+import * as animations from '../../styles/animations.css';
+
+export interface TitlePaneProperties extends ThemeableProperties {
+	ariaHeadingLevel?: number;
+	closeable?: boolean;
+	enterAnimation?: string;
+	exitAnimation?: string;
+	open?: boolean;
+	title: string;
+
+	onRequestClose?(): void;
+	onRequestOpen?(): void;
+};
+
+@theme(css)
+export default class TitlePane extends ThemeableMixin(WidgetBase)<TitlePaneProperties> {
```

Comment:
We need to set the result of `ThemeableMixin(WidgetBase)` to a `const` and use the result to extend from.

```ts
const TitlePaneBase = ThemeableMixin(WidgetBase);

export default class TitlePane extends TitlePaneBase<TitlePaneProperties> {
```